libs/game/info.ts

```diff
@@ -383,6 +383,26 @@ namespace info {
         infoState.countdownExpired = true;
     }
 
+    /**
+     * Get the value of the current count down
+     */
+    //% block="countdown"
+    //% blockId=gamegetcountdown
+    //% group="Countdown"
+    export function getCountdown() {
+        return (infoState.gameEnd - game.currentScene().millis()) / 1000;
+    }
+
+    /**
+     * Get the value of the current count down
+     */
+    //% block="change countdown by $seconds (s)"
+    //% blockId=gamechangecountdown
+    //% group="Countdown"
+    export function changeCountdownBy(seconds: number) {
+        infoState.gameEnd += (seconds * 1000);
```

Comment:
This should probably call into startcountdown so the flags get set - changeCountdownBy(5) when no countdown is set should be the same as a startCountdown(5)